serial_monitor_to_file.py

```diff
@@ -0,0 +1,46 @@
+#imports
+import serial.tools.list_ports
+from datetime import datetime
+
+#list out all ports in use
+ports = serial.tools.list_ports.comports()
+serial_instance = serial.Serial()
+port_list = []
+port_variable = ""
+
+for port in ports:
+    #append each port to the list of ports
+    port_list.append(port)
+    #print out all ports
+    print(port)
+
+
+value = input("select Port: COM")
```

Comment:
```suggestion
value = f'COM{input("Select Port: COM")}'
```

Reply:
hvad gør f'com ?

Reply:
Jeg kan godt se den ser lidt tricky ud.
Jeg bruger bare string interpolation til at assign input + COM til value.
Så brugeren ser: Select Port: COM_, 
hvorefter de skriver et tal, der bliver fanget af `input` funktionen, der er i string assignment til value.

Så lad os sige brugeren skriver 3, så vil `value` have værdien COM3.

Reply:
Jeg så bare du hele tiden tilføjede COM + value før, så jeg lavede bare den her assignment, så det kun skal gøres én gang.

Reply:
yes i get it, det er fixed